CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 ## v1.0.0
 
 ### General
+- Code has been made python 3 compatible. [[#804]](https://github.com/aiidateam/aiida_core/pull/804)[[#2136]](https://github.com/aiidateam/aiida_core/pull/2136)[[#2125]](https://github.com/aiidateam/aiida_core/pull/2125)[[#2117]](https://github.com/aiidateam/aiida_core/pull/2117)[[#2110]](https://github.com/aiidateam/aiida_core/pull/2110)[[#2100]](https://github.com/aiidateam/aiida_core/pull/2100)[[#2094]](https://github.com/aiidateam/aiida_core/pull/2094)[[#2092]](https://github.com/aiidateam/aiida_core/pull/2092)
+- All files written by AiiDA. [[#2107]](https://github.com/aiidateam/aiida_core/pull/2107)
```

Comment:
```suggestion
- All files & databases written by AiiDA are now in UTF-8. [[#2107]](https://github.com/aiidateam/aiida_core/pull/2107)
```

Reply:
I wouldn't delete this, I think it's very important.

Reply:
I am still not a hundred percent sure about whether this really includes all files. Are all files in the repository also always written as utf-8, i.e. also those retrieved from for a job calculation from a remote computer? If those files on the remote are in a different encoding, do we convert them to utf-8?

Reply:
Happy to rephrase (it definitely does not include general binary files, which might not even have a text representation).
You could say something like  "AiiDA now enforces UTF-8 for text output in its files and databases"

Reply:
I have added a slightly rephrased line (or adapted the line I actually forgot to delete :)